index.html

```diff
@@ -51,50 +51,57 @@
     <div class="container main-content" >
         <!-- row 1 -->
         <div id="content-search" class="row">
-            <div class="col-sm-4">
-                Text in row 1 column 1
+            <div id="row1-column1" class="col-sm-4">
+                <img id="card-image" src="">
+                
             </div>
-            <div class="col-sm-4" >
-    
+            <div id="row1-column2" class="col-sm-4" >
+              <div id="card-name-after-searched"> </div>
```

Comment:
card-name is much better id name for this.   the **after-searched** part seems extraneous.